samples/openthread/cli/src/low_power.c

```diff
@@ -8,7 +8,9 @@
 #include <zephyr/net/openthread.h>
 #include <zephyr/device.h>
 #include <zephyr/pm/device.h>
+#if defined(CONFIG_RAM_POWER_DOWN_LIBRARY)
```

Comment:
Isn't it always enabled for the lowpower snippet?

Reply:
It is, but if you would like to disable only the `CONFIG_RAM_POWER_DOWN_LIBRARY`, for example using a CMAKE argument, and still use the low_power snippet it will not be possible. It is required for testing with and without disabling a part of RAM to compare results.